### PR TITLE
Add support for autocompletion of tasks

### PR DESCRIPTION
**Pull request by mbrodala**
Notice that this requires the "runner" command to be installed globally and executed as such.

### CODE REVIEW

File: src/cli.sh [hunk 1]
```diff
@@ -113,3 +122,6 @@ fi
 ## Source the runnerfile
 # shellcheck source=/dev/null
 source "${runner_file}"
+
+## Parse the actual arguments
+runner_cli_parse_args "${runner_args[@]}"
```
Comment (mbrodala) [inline src/cli.sh]: This move was necessary since we need to load the runnerfile to get the list of defined tasks.

Comment (stylemistake) [inline src/cli.sh]: But then `-f` option won't work.

Comment (stylemistake) [inline src/cli.sh]: What if you store an **intent** to show the list of tasks (in some variable) when parsing the arguments, and then after proper initialization you handle the task listing?

**Comment by stylemistake** [comment]
Issue #11

File: src/cli.sh (hunk 1 above)
Comment (mbrodala) [inline src/cli.sh]: Slightly workaround-ish but it might work. I'll give it a try.

**Comment by stylemistake** [comment]
> Notice that this requires the "runner" command to be installed globally and executed as such.
It is possible to generate some custom completion code, with custom paths and such, which you would then eval and get it working.

**Comment by stylemistake** [comment]
Strange, Github should've marked this as merged (I did a cherry-pick).

**Comment by mbrodala** [comment]
Thanks for merging. Will you update the README or shall I do a PR for that?

**Comment by stylemistake** [comment]
> shall I do a PR for that

Would be great.

**Pull request closed without merging**